core/build.gradle

```diff
@@ -46,6 +46,8 @@ dependencies {
     compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.10'
     compile group: 'com.facebook.presto', name: 'presto-matching', version: '0.240'
     compile group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'
+    compile group: 'org.opensearch.ml', name:'opensearch-ml-client', version: '1.3.0.0'
+    compile group: 'org.opensearch', name: 'opensearch', version: "1.3.0-SNAPSHOT"
```

Comment:
Core module should not depend on specific storage engine.

Reply:
Yes it shouldn't be here, I forget to clear it here.

Reply:
Attempted to move ml-commons dependency and opensearch dependency to opensearch module. However, it causing jar hell exception while installing opensearch1.3.

Reply:
Will this PR https://github.com/opensearch-project/ml-commons/pull/137 fix the jar hell issue?

Reply:
yes, after it's merged, I no longer see the jar hell issue in build.